samples/bot-formatting-cards/csharp/BotFormattingCards/Resources/adaptiveCardWithEmoji.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.0",
```

Comment:
Can't we use latest version?

Reply:
I have updated 1.5.
Teams platform supports v1. 5 or earlier of Adaptive Card.